Sources/UIScreen+render.swift

```diff
@@ -91,14 +91,34 @@ extension UIScreen {
     }
 
     func outline(_ rect: CGRect, lineColor: UIColor, lineThickness: CGFloat) {
+        // we want to render the outline 'inside' the rect rather
+        // than exceeding the bounds when lineThickness is bigger than 1
```

Comment:
![IMG_3302 2](https://user-images.githubusercontent.com/5617793/107775815-19ea0100-6d41-11eb-8895-db0ab6b66f14.jpg)

When using a bigger lineThickness the rect (grey) which renders the borderWidth gets centered along the original bounds of the base rect (blue). Thats why it makes sense to move by offset `let offset = lineThickness / 2`. But I do not fully get how this works with borderdWidths of 2 without retina. It always looked correct in my experiments.

@mouselangelo

Reply:
Ah ok - so basically by default the border is "centered" but we want it to offset so that it's "inside". Makes sense now.
Thanks for the explanation - the drawing helps too :)